app/src/androidTest/java/com/github/se/cyrcle/ui/report/AdminScreenTest.kt

```diff
@@ -156,4 +176,65 @@ class AdminScreenTest {
     composeTestRule.onNodeWithTag("ReportCard0").performClick()
     composeTestRule.onNodeWithTag("CheckReportsButton0").performClick()
   }
+
+  @Test
+  fun testAlertDialogAppearsOnMoreOptionsClick() {
+    composeTestRule.setContent {
+      AdminScreen(
+          navigationActions = navigationActions,
+          reportedObjectViewModel = reportedObjectViewModel,
+          parkingViewModel = parkingViewModel,
+          reviewViewModel = reviewViewModel)
+    }
+    composeTestRule.onNodeWithTag("ReportCard0").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNodeWithTag("MoreOptionsButton0").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNode(isDialog()).assertIsDisplayed()
+    composeTestRule.onNodeWithText("Details for this Reported Object:").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Loading parking details...").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Go To Parking").assertIsDisplayed().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").assertIsDisplayed().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNode(isDialog()).assertIsNotDisplayed()
+  }
+
+  @Test
+  fun testReportDetailsContentForParkingType() {
+    composeTestRule.setContent {
+      ReportDetailsContent(
+          objectType = ReportedObjectType.PARKING,
+          objectUID = parking1.uid,
+          userUID = parking1.owner,
+          reviewViewModel = reviewViewModel,
+          parkingViewModel = parkingViewModel)
+    }
+    composeTestRule
+        .onNodeWithTag("BulletPointParkingUID")
+        .assertIsDisplayed()
+        .onChildren()
+        .assertAny(hasText("Parking UID: Test_spot_1"))
```

Comment:
That is just a minor detail but couldn't you use `assertText` for example? Maybe the name is not exact but there exists an assert which aim to verify the text contained inside a composable.

Reply:
Depends on what you mean by problem. If you have trouble implementing it now it is not a problem but it would be better to implement it from my point of view.

Reply:
Done, in the same occasion I made the tests a little simpler. No need to reinvent the wheel since it's just checking that specific tests are displayed